content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md

```diff
@@ -69,11 +70,69 @@ For a complete list of supported libraries and processor architectures (includin
 
 ### Installation
 
-1. [Install the tracer.](#install-the-tracer)
-2. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
-3. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+1. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+2. [Install the tracer.](#install-the-tracer)
+3. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
 4. [View your live data.](#view-your-live-data)
 
+### Configure the Datadog Agent for APM
+
+[Install and configure the Datadog Agent][2] to receive traces from your instrumented application. By default, the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic on `http://localhost:8126`.
+
+For containerized, serverless, and cloud environments:
+
+{{< tabs >}}
+
+{{% tab "Containers" %}}
+
+1. Set `apm_non_local_traffic: true` in the `apm_config` section of your main [`datadog.yaml` configuration file][1].
+
+2. See the specific setup instructions to configure the Agent to receive traces in a containerized environment:
+
+{{< partial name="apm/apm-containers.html" >}}
+</br>
+
+3. After instrumenting your application, the tracing client sends traces to `localhost:8126` by default. If this is not the correct host and port, change it by setting the `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` environment variables. For more information on configuring these settings, see [Configuration](#configuration).
```

Comment:
Same remark as in the other PR, this is Tracer configuration. It should be located in a Tracer section